### PR TITLE
fix: do not build unneeded utilities and man for SELinux libraries

### LINKED ISSUE #1067: Minimize SELinux libraries, drop unneeded binaries
We should likely drop all these binaries likely being installed together with libsepol or libselinux.

```
#288 0.200 Relabeled /rootfs/usr/sbin/avcstat from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/compute_av from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/compute_create from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/compute_member from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/compute_relabel from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/getconlist from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/getdefaultcon from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/getenforce from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/getfilecon from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/getpidcon from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/getpidprevcon from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/getpolicyload from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/getsebool from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/getseuser from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/matchpathcon from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/policyvers from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/sefcontext_compile from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/selabel_digest from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/selabel_get_digests_all_partial_matches from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/selabel_lookup from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/selabel_lookup_best_match from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/selabel_partial_match from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/selinux_check_access from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/selinux_check_securetty_context from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/selinuxenabled from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/selinuxexeccon from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/setenforce from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/setfilecon from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/togglesebool from <no context> to system_u:object_r:bin_t:s0
#288 0.200 Relabeled /rootfs/usr/sbin/validatetrans from <no context> to system_u:object_r:bin_t:s0
```

### PR DESCRIPTION
**Pull request by dsseng**
Utilities end up on rootfs increasing size and binary count

Fixes #1067

Signed-off-by: Dmitry Sharshakov <dmitry.sharshakov@siderolabs.com>

### CODE REVIEW

**Review by frezbo — APPROVED** [review]

File: libsepol/patches/dont-build-utils.patch [hunk 1]
```diff
@@ -0,0 +1,31 @@
+diff --git a/Makefile b/Makefile
+index 5b5d03e2..5744d7a4 100644
+--- a/Makefile
++++ b/Makefile
+@@ -4,26 +4,21 @@ export DISABLE_CIL
+
```
Comment (smira) [inline libsepol/patches/dont-build-utils.patch:6]: if we don't want to maintain patches, we can also let it build, but then do `rm -rf /rootfs/usr/....`

**Review by smira — APPROVED** [review]

File: libsepol/patches/dont-build-utils.patch (hunk 1 above)
Comment (dsseng) [reply libsepol/patches/dont-build-utils.patch:6]: They aren't likely to change that often. Maybe I should send a patch upstream so there's an option like SUBDIRS in libselinux

**Comment by smira** [comment]
@dsseng should we rebase/merge this?

**Comment by dsseng** [comment]
> @dsseng should we rebase/merge this?

Should we keep the patch or delete built stuff? See my response. After closing that thread it's good to go

**Comment by smira** [comment]
> > @dsseng should we rebase/merge this?
> 
> Should we keep the patch or delete built stuff? See my response. After closing that thread it's good to go

up to you, it's your call

File: libsepol/patches/dont-build-utils.patch (hunk 1 above)
Comment (dsseng) [reply libsepol/patches/dont-build-utils.patch:6]: Upstreamed a couple of change for SUBDIRS in every lib.

**Pull request merged**